### DISCUSSION TITLE
AMDC Rev F: alternate parts proposal for the components not available in stock

### DISCUSSION THREAD

**Discussion by DivyaMendpara** [Severson-Group/AMDC-Hardware#229] [General]
Hello, @elsevers , @npetersen2 and @codecubepi 

I am attaching excel sheet for AMDC Rev F BOM. In the second sheet I have proposed alternate part numbers for the components which are not available on Digikey currently. Kindly review and provide your feedback. 

[BOM_RevF.xlsx](https://github.com/Severson-Group/AMDC-Hardware/files/12872786/BOM_RevF.xlsx)

**Comment by npetersen2** [comment]
@DivyaMendpara I am taking a look at your sheet and will respond soon with my feedback

**Comment by npetersen2** [comment]
@DivyaMendpara thanks for looking into this!

I went through and checked all the parts you listed. I approve 5 of 6 of the alternative parts. However, 1 of them will not work with the current PCB layout. We should discuss this one, the `-15V 500mA linear regulator`

Here is your sheet with my comments added: [BOM_RevF.xlsx](https://github.com/Severson-Group/AMDC-Hardware/files/13030720/BOM_RevF.xlsx)

**Comment by npetersen2** [comment]
@codecubepi if you have some time, can you check the alternative parts that @DivyaMendpara suggested and confirm my review?

**Comment by codecubepi** [comment]
@DivyaMendpara 

[BOM_RevF.xlsx](https://github.com/Severson-Group/AMDC-Hardware/files/13034984/BOM_RevF.xlsx)

I added my comments, I agree with @npetersen2

**Comment by elsevers** [comment]
@DivyaMendpara, when you order boards, can you post the final version of the alternate parts to this issue?

**Reply by DivyaMendpara** [reply]
Hello @elsevers, alternate part numbers used while placing the order in screaming circuits are attached in this excel sheet
[RevF_altparts_Screamingckt.xlsx](https://github.com/Severson-Group/AMDC-Hardware/files/13310162/RevF_altparts_Screamingckt.xlsx)

**Reply by elsevers** [reply]
Thanks @DivyaMendpara

**Comment by elsevers** [comment]
From @DivyaMendpara, here is the Screaming Circuits project summary
[ProjectSummary.pdf](https://github.com/Severson-Group/AMDC-Hardware/files/13310454/ProjectSummary.pdf)

and list of parts they are ordering
[8481-3533_AMDC_Giri_4b3a7bcfb6be.xlsx](https://github.com/Severson-Group/AMDC-Hardware/files/13310459/8481-3533_AMDC_Giri_4b3a7bcfb6be.xlsx)

**Reply by elsevers** [reply]
Missing parts (Screaming Circuits will not populate) are

- the PicoZed because it was out of stock, but was [just restocked at avnet](https://www.avnet.com/shop/us/products/avnet-engineering-services/aes-z7pz-7z030-som-i-g-rev-e-3074457345635303359) 
- U19 / MC79M15CDTRKG  – the only approved vendor for Screaming Circuits they could find has a lead time of 13 weeks, so we decided to populate this ourselves too. Digikey & Mouser have a ship date estimate of Sept 2024
[Winsource](https://www.win-source.net/products/detail/on-semiconductor/mc79m15cdtrkg.html?utm_source=findChips&utm_medium=buyNow) says it has stock / can ship immediately from HK warehouse
